src/Marketplace/NotificationTargetController.php

```diff
@@ -78,6 +78,11 @@ public function addDataForTemplate($event, $options = [])
             ];
         }
 
+        $this->data['##glpi.url##'] = $this->formatURL(
+            NotificationTarget::GLPI_USER,
```

Comment:
Not sure of this one, but all calls to `formatUrl` are using the same value
```suggestion
            $options['additionnaloption']['usertype'],
```